models/dodo/optimism/dodo_optimism_sources.yml

```diff
@@ -0,0 +1,109 @@
+version: 2
+
+sources:
+  - name: dodo_optimism
+    description: >
+      Decoded tables related to Dodo dex trades for Optimism
+    freshness: # default freshness
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    tables:
+      - name: DODO_evt_SellBaseToken
+        description: >
+          Decoded table related to the Base Token sell events version 1.
+        loaded_at_field: evt_block_time
+        columns:
+          - name: contract_address
+            description: 'Address for the liquidity pool used in transaction'
+          - name: evt_tx_hash
+            description: 'Transaction hash of the event'
+          - name: evt_index
+            description: 'Event index'
+          - name: evt_block_time
+            description: 'Timestamp for Sell event time in UTC'
+
+      - name: DODO_evt_BuyBaseToken
+        description: >
+          Decoded table related to Base Token buy events version 1.
+        loaded_at_field: evt_block_time
+        columns:
+          - name: contract_address
+            description: 'Address for the liquidity pool used in transaction'
+          - name: evt_tx_hash
+            description: 'Transaction hash of the event'
+          - name: evt_index
+            description: 'Event index'
+          - name: evt_block_time
+            description: 'Timestamp for Buy event time in UTC'
+
+      - name: DVM_evt_DODOSwap
+        description: >
+          Decoded table related to the DVM DODOSwap contract events.
+        loaded_at_field: evt_block_time
+        columns:
+          - name: contract_address
+            description: 'Address for the liquidity pool used in transaction'
+          - name: evt_tx_hash
+            description: 'Transaction hash of the event'
+          - name: evt_index
+            description: 'Event index'
+          - name: evt_block_time
+            description: 'Timestamp for block event time in UTC'
+
+      - name: DPP_evt_DODOSwap
```

Comment:
Seems this source cannot be found:
https://dune.com/queries?d=1&blockchain=optimism&category=decoded_project&namespace=dodo

Reply:
Have already decoded DPP contract on optimism. And fix the error